toolkits/nature/packages/nature-sort-by/README.md

```diff
@@ -25,34 +25,17 @@ sortBy();
 ```scss
 @import '@springernature/nature-sort-by/scss/10-settings/sort-by';
 @import '@springernature/nature-sort-by/scss/50-components/sort-by';
-
-@import '@springernature/brand-context/default/scss/60-utilities/flex.scss';
-@import '@springernature/brand-context/default/scss/60-utilities/hiding.scss';
-@import '@springernature/brand-context/default/scss/60-utilities/spacing.scss';
-
 ```
 
-> **NOTE** The component require the use of the utility classes shown above
+The component's HTML should be inserted inside of an HTML form in your application. See below [Template section](#template) for an example.
```

Comment:
Will the anchor link here work on the Elements website? @sangitamane any idea?

Reply:
Thanks Amy. This approach for hash linking in the README markdown files seems to be what we have done across all Elements components and brand contexts. So perhaps if we need to change that in any way we do it separately in another PR and we do across the entire Frontend Toolkits codebase?

![image](https://user-images.githubusercontent.com/5796370/154047456-f26d6b0f-2972-4f5c-9537-895b7b105fa3.png)